packages/core/bootstrap/src/lib/middleware/cache/index.ts

```diff
@@ -18,7 +18,7 @@ import * as redis from './redis'
 import * as metrics from './metrics'
 import { CacheOptions, CacheImplOptions, CacheEntry, Cache } from './types'
 
-const DEFAULT_CACHE_KEY_GROUP = uuid()
+const UUID = uuid()
```

Comment:
If this isn't used anywhere can we remove it?

Reply:
Is is very unlikely, but still possible that there will be no name for adapter and UUID is the fallback

Reply:
I thought EAs were force to have a name, but I also don't trust it enough haha. Even in that case we can take care of this in a separate PR